hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmVolumeArgs.java

```diff
@@ -84,6 +97,14 @@ private OmVolumeArgs(String adminName, String ownerName, String volume,
     this.updateID = updateID;
   }
 
+  public long getRefCount() {
+    assert (refCount >= 0);
```

Comment:
Ah. I added this `assert` for potential debugging use. Since Java asserts are ignored in production.

I could change this into `Preconditions.checkState`.